TradeItIosTicketSDK2/TradeItSelectBrokerViewController.swift

```diff
@@ -219,4 +222,18 @@ class TradeItSelectBrokerViewController: CloseableViewController, UITableViewDel
 
         return cell
     }
+
+    private func getRemoteLogo(forBroker broker: TradeItBroker, imageView: UIImageView) -> Bool {
```

Comment:
Not really because `getLogo` is sync but this needs to be `async` and also needs the data from the `getBrokerList` call